tutorial2_quantities_and_amounts/README.md

```diff
@@ -0,0 +1,59 @@
+# Tutorial 2
+
+The goals for this tutorial are:
+
+- to get comfortable with the [Github workflow](../GIT_SETUP.md), and with using their [local development environment](../README.md).
+- to understand how to load in a dataset using [d3-fetch](https://github.com/d3/d3-fetch) (d3.json, d3.csv, etc.).
+- to introduce [HTML svg](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg) coordinate system and how [d3-scales](https://bost.ocks.org/mike/selection/) can place graphical primitives on the page.
+- to practice creating a simple bar chart.
+
+## How to setup, serve, and deploy:
+
+From tutorial 1, you should have all the correct [installations](../README.md#setup), and have set up your [git flow](../GIT_SETUP.md) (forking, cloning, setting upstream, etc.). 
+
+For this tutorial, you will **still need** to get a [basic server](../BASIC_SERVER.md) up and running. Use your terminal (or git bash, on PC) to navigate to the _root directory_ of your fork. Your path may look different, but the command will be something like this:
+
+```sh
+$ cd Documents/My\ Repositories/Interactive-Data-Vis-Sp2020/
+```
+
+From this level, (_not from the tutorial folder_), run:
+
+```sh
+$ serve
+``` 
+
+When you open `http://localhost:5000/`, you should see your repository's folder structure. 
+
+You can view your **tutorial site** at `http://localhost:5000/TUTORIAL_PATH/`. 
+
+You can view our class **demo site** at `http://localhost:5000/TUTORIAL_PATH/demo/`.
+
+(note: make sure to change `TUTORIAL_PATH`). 
+
+As you're working on the assignment, you can always reference the [demo code](demo/).
+
+### Deploy
+
+Once you've completed the assignment, use the Github workflow to deploy your work to **your fork** of the course repository. Post the link to your deployed example as a comment to the appropriate post on the Commons. (Note: your link will look something like: `https://YOUR_USERNAME.github.io/Interactive-Data-Vis-Sp2020/TUTORIAL_PATH/`)
+
+To receive full credit, you must post your stable path before the start of the next class.
+
+## Week 2 Assignment:
+
+- [ ] Replicate the visual for either the same dataset, or a new one. As you do each week, you should leverage the files present in the root of this directory (`tutorial1_intro/` [index.html](index.html), [style.css](style.css), [main.js](main.js))
```

Comment:
Some of my students got confused what "replicate the code" meant -- maybe we should write something like:
 "implement your own bar chart in the files present in the root of this directory... You may use an existing dataset or a new one, the choice is yours"